clang/test/CIR/Lowering/ThroughMLIR/compilation.c

```diff
@@ -0,0 +1,8 @@
+// RUN: %clang -fclangir -fno-clangir-direct-lowering %s -o %t
+
+int main(int argc, char *argv[]) {
```

Comment:
Can you tide this up a bit? 2 spaces + rename `main` to some other function name.

Reply:
I still see 4 space identation.